lib/pure/times.nim

```diff
@@ -880,6 +880,11 @@ since((1, 1)):
   export fromUnixFloat
   export toUnixFloat
 
+
+when defined(windows): # TODO move to winlean
```

Comment:
If this should be in winlean, why not move it now?

Reply:
winlean contains many wrong definition s. I'm not sure the future of winlean.

Reply:
see https://github.com/nim-lang/Nim/issues/12327

Reply:
a PR doesn't have to solve everything, the minimum bar is improving one aspect while not making anything worse and not preventing any future good design.

TODO's are good, it helps the reader see what should be addressed in future work, and yes, they're regularly addressed in cleanup PRs

Reply:
> winlean contains many wrong definition s. I'm not sure the future of winlean.
> 
> I could remove Todo instead.

Do this incorrect definitions prevent moving this procedure?

Reply:
All right.